### PR TITLE
🐛 Quick fix for DOM parsing and remove likertemp

### PR DESCRIPTION
**Pull request by williamchong**
- Fix DOM parsing error due to youtube change
- Remove likertemp for now since it does not do anything now and quite spammy

### CODE REVIEW

File: src/ContentScript/injectors/youtube.plugin.ts [hunk 1]
```diff
@@ -78,26 +78,23 @@ class YoutubePlugin {
...
-    if (!nodes || nodes?.length === 0) {
-      return '';
-    }
-    const node = nodes[nodes.length - 1];
-    const url = node.innerText;
-    if (likeCoTest.test(url)) {
-      id = url.split('/')[url.split('/').length - 1]
-    }
-    return 'id';
+    const likeCoTest = new RegExp('https://button.like.co/([a-z0-9-_]{7,20})');
```
Comment (nnkken) [inline src/ContentScript/injectors/youtube.plugin.ts:88]: `.` in Regexp needs escape, otherwise it matches all characters (e.g. `buttonilike.co`)

**Review by nwingt — APPROVED** [review]

File: src/ContentScript/injectors/youtube.plugin.ts [hunk 2]
```diff
@@ -98,6 +106,11 @@ class YoutubePlugin {
   }
 
   private insertLikeCoinButton(likerId: string) {
+    if (!likerId) {
+      const b = document.querySelector('.button-container');
```
Comment (nnkken) [inline src/ContentScript/injectors/youtube.plugin.ts:110]: 🤔This class name looks quite general...
Is it possible to change to something like `like-button-container` so it won't break Youtube functionality when there's change on Youtube site in the future?

File: src/ContentScript/injectors/youtube.plugin.ts [hunk 3]
```diff
@@ -85,7 +85,7 @@ class YoutubePlugin {
   }
 
   private getLikeId() {
-    const likeCoTest = new RegExp('https://button.like.co/([a-z0-9-_]{7,20})');
+    const likeCoTest = new RegExp(/https:\/\/button\.like\.co\/([a-z0-9-_]{7,20})/);
```
Comment (nwingt) [inline src/ContentScript/injectors/youtube.plugin.ts:96]:
```suggestion
    const likeCoTest = /https:\/\/button\.like\.co\/([a-z0-9-_]{7,20})/
```

File: src/ContentScript/injectors/youtube.plugin.ts [hunk 4]
```diff
@@ -119,7 +108,7 @@ class YoutubePlugin {
     while (ele.lastElementChild) {
       ele = ele.lastElementChild;
     }
-    const btn = ele.querySelector('.button-container');
+    const btn = ele.querySelector('.like-button-container');
     if (btn) return;
     const buttonContainer = document.createElement('div');
     buttonContainer.className = 'button-container';
```
Comment (nnkken) [inline src/ContentScript/injectors/youtube.plugin.ts:114]: Need to change className to `like-button-container` here

**Pull request merged**